benchmark.py

```
@@ -0,0 +1,838 @@
+            # This simulates a query that contains the fingerprint as a subset
+            additional_count = self.rng.randint(0, 20)  # 0-20 additional random hashes
+            for _ in range(additional_count):
+                # Add random hash that might not exist in the fingerprint
+                exact_subset.append(self.rng.randint(0, self.config.max_hash_value))
+            
+            # Debug: Print info about the search being generated
+            if len(self.inserted_ids) <= 10:  # Only print for small datasets
+                print(f"    DEBUG: Generating match search from FP {fp_id}, using {subset_size}/{len(base_hashes)} hashes + {additional_count} random")
+            
+            return exact_subset
+        else:
```

Ruff (0.12.2)

Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)